android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -247,46 +260,62 @@ constructor(
           .runCatching {
             // Show Progress indicator while evaluating measure
             toggleProgressIndicatorVisibility(true)
-
-            withContext(dispatcherProvider.io()) {
-              fhirEngine.loadCqlLibraryBundle(fhirOperator, measureUrl)
-            }
-
-            if (reportTypeSelectorUiState.value.patientViewData != null && individualEvaluation) {
-              val measureReport: MeasureReport? =
-                try {
-                  withContext(dispatcherProvider.io()) {
-                    fhirOperator.evaluateMeasure(
-                      measureUrl = measureUrl,
-                      start = startDateFormatted,
-                      end = endDateFormatted,
-                      reportType = SUBJECT,
-                      subject = reportTypeSelectorUiState.value.patientViewData!!.logicalId,
-                      practitioner =
-                        practitionerId?.asReference(ResourceType.Practitioner)?.reference,
-                      lastReceivedOn = null // Non-null value not supported yet
-                    )
+            measureReportConfigList.forEach {
```

Comment:
Just a question, how  will this impact performance? evaluating all the measures in the list. We can improve it later on.

Reply:
Yes, generating reports for all the indicators in single module will impact performance,Can you suggest other way around ?